autorest/adal/sender.go

```diff
@@ -0,0 +1,46 @@
+package adal
```

Comment:
What is the purpose of this? Does ADAL need it's own Sender impl? Can it not use the standard one used elsewhere?